ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -50,7 +50,7 @@ const isValidFacetAttribute = (id, type) => {
   if (id === 'anyText') {
     return false
   }
-  if (!properties.facetWhitelist.includes(id)) {
+  if (!properties.attributeSuggestionList.includes(id)) {
```

Comment:
just to confirm, this is the only area on the front end that uses the `facetWhitelist` property?

Reply:
Yep